Cargo.toml

```diff
@@ -19,14 +19,14 @@ exclude = [
 [dependencies]
 log = "0.4"
 imgui = "^0.3"
-ash = ">=0.29"
```

Comment:
You should let >=0.29 here, since 0.31 breaks none of the apis we use in this project that will work just fine :) Just tested in another projet : https://github.com/adrien-ben/gltf-viewer-rs/commit/1ca02925c91e8339e43e9042f91c40b9d031a118

Reply:
Done.